### DISCUSSION TITLE
Cant Login Into BTCpay After changing email

### DISCUSSION THREAD

**Discussion by ItchyPong** [btcpayserver/btcpayserver#2231] [Technical Support] [answered]
Hi all hope you can help :) I recently set up BTCpay for woocommerce. But i changed my BTCpay email and now i cant login into btcpay . I have tried the forgot password multiple times and checked my spam but nothing is arriving. I am using proton mail now is this an issue?   

Really need to get into my account thanks for help

**Comment by pavlenex** [comment, marked as answer]
Hi @ItchyPong what is the link to your btcpay server or are you using a third-party host?

**Reply by ItchyPong** [reply]
I am using lunanode the link is https://btcpay.vitalherbs.co.uk/  

thanks for reply

**Reply by ItchyPong** [reply]
Any one? I am now totally locked out of BTCpay , forgot password with old and new email address is doing nothing. No emails in spam or inbox of either?

**Reply by pavlenex** [reply]
To get the password via email, you first needed to set up an SMTP server. The reason you're not receiving it, is because you haven't set it up. Now, to retrieve a password or email follow the instructions here https://docs.btcpayserver.org/FAQ/FAQ-ServerSettings/#forgot-btcpay-admin-password (if you're the owner of this server you provided).

**Reply by ItchyPong** [reply]
Thanks for the help is this in proton mail ? or Lunanode server?

**Reply by pavlenex** [reply]
This is on your btcpay server (in your case LunaNode).

**Reply by ItchyPong** [reply]
Ok I have tried to access vnc console to input the code but I am getting asked to login into it and the password is not working there ??

**Reply by pavlenex** [reply]
This is all already well-documented in our docs, so check these:
https://docs.btcpayserver.org/FAQ/FAQ-ServerSettings/#how-to-ssh-into-my-btcpay-running-on-vps
https://github.com/JeffVandrewJr/patron/blob/master/SSH.md

**Reply by ItchyPong** [reply]
This is far beyond what my brain can handle !   Is there a easier work around eg delete server and start whole proccess again?

**Reply by ItchyPong** [reply]
Ok I give up this is beyong what i can do I cant login into any console the steps to do so start with find initial login details on lunanode...I dont even see that ? There is no intial login details ??

**Reply by pavlenex** [reply]
I'm unsure what you're asking. To login into your btcpay server and apply commands, you need to log in into your VPS. The login server credentials are provided by lunanode. I can't provide any further guidance than this, it's all documented already, if you need more help join our chat.btcpaysever.org perhaps someone else can hand-hold you during the process.
